plutarch-test/common/Plutarch/Test/Golden.hs

```diff
@@ -64,6 +70,21 @@ data GoldenValue = GoldenValue
   -- ^ User test's expectation function
   }
 
+data GoldenConf = GoldenConf
+  { trackPreEval :: Bool
+  -- ^ Whether to track UPLC golden.
+  , trackPostEval :: Bool
+  -- ^ Whether to track evaluated UPLC golden.
+  , trackBench :: Bool
+  -- ^ Whether to track benchmark golden.
```

Comment:
Any reason to have these 3 separate fields instead of a single `Set GoldenTest` field?

Reply:
Yup, I'd say that goes without saying.

Maybe: `data GoldenTest = GoldenTest_Bench | ...`.